utils/vamm-math/VammConfiguration.sol

```diff
@@ -22,6 +22,14 @@ library VammConfiguration {
         UD60x18 spread;
         /// @dev rate oracle from which the vamm extracts the liquidity index
         IRateOracle rateOracle;
+        /// @dev The minimum tick of the vamm
+        int24 minTick;
+        /// @dev The maximum tick of the vamm
+        int24 maxTick;
+        /// @dev Equivalent to getSqrtRatioAtTick(MAX_TICK)
+        uint160 minSqrtRatio;
```

Comment:
Should we put them outside of configuration? To the state of VAMM as we've talked during the chat?